System-Architecture-Overview.md

```diff
@@ -0,0 +1,40 @@
+# Zadig System Architecture Overview
+
+## System Architecture
+
+![Architecture_diagram](./Zadig-System-Architecture.svg)
+
+## Main Components
+
+User Interface：
+- zadig-portal：zadig web component
+- kodespace：zadig command line tools
+- Zadig Toolkit：vscode plugin
+
+API Gateway：
+- Gloo Edge：Zadig API gateway
+- OPA：Authentication and authorization
+- User：User management, token generation
+- Dex：Identity service for Zadig, acts as a portal to other identity providers like AD, LDAP, OAuth2, GitHub, ...
+
+Zadig Core：
+- Picket：backend for frontend service.
+- Policy:  data source of OPA and policy registration center.
+- Aslan：main service for all business logic. Project, environment, service, workflow, build, system management are all in this service.
```

Comment:
`Project, environment, service, workflow, build, system management are all in this service.` -> `Project /  environment / service / workflow / build / system management are all in this service.`